setup.py

```diff
@@ -429,6 +429,18 @@ def macos_extensions():
 
 
 if __name__ == "__main__":
+
+    msg = ("SWIG is a required build dependency of Enable. Furthermore, there "
+          "is currently a known issue with SWIG 4.0, see enthought/enable#360."
+          " Please download SWIG 3.0.x (see http://www.swig.org/).")
+    try:
+        with subprocess.Popen(["swig", "-version"], stdout=subprocess.PIPE, encoding='utf-8') as proc:
+            swig_version_line = proc.stdout.read().split('\n')[1]
+            swig_version = swig_version_line.split(' ')[2]
+            if  not swig_version.startswith('3'):
```

Comment:
I guess as #360 mentions swig 4 breaks Agg backend specifically.  Maybe this should just be a warning here instead of raising?  Saying beware agg won't work but if you wanna use non agg, go right ahead as you are?

Reply:
> Maybe this should just be a warning here instead of raising?

I think this should be an error. Warnings during installation might be ignored.